### ISSUE TITLE
All upload buttons have same text when read by screen reader

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#369]
All 3 upload buttons, upload file, upload folder and multiple file upload, have same text when read by screen reader and so cannot differentiate 
![screen shot 2016-07-06 at 11 42 27](https://cloud.githubusercontent.com/assets/13200338/16615498/03d5e2e0-436f-11e6-8712-acfd86cd8160.png)

**Comment by eromano** [comment]
Now we have only on button
